### PR TITLE
[Docs] Revise intro to match how sequin works, update diagram

### PR DESCRIPTION
**Pull request by thisisgoldman**

### CODE REVIEW

**Comment by thisisgoldman** [comment]
* **#391** <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/391?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @thisisgoldman and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: docs/how-sequin-works.mdx [hunk 1]
```diff
@@ -5,10 +5,10 @@ icon: "wrench"
 iconType: "solid"
 ---
 
-With Sequin, you create [**sequences**](#sequences) which present a strictly ordered view of rows from one or more tables. Then, you can use this sequence to consume a table's rows by:
+With Sequin, you enable Postgres to stream data by creating [**sequences**](#sequences) which present a strictly ordered view of rows from one or more tables. Then, you can use this sequence to consume a table's rows by:
```
Comment (thisisgoldman) [inline docs/how-sequin-works.mdx:8]: Small edit to make it clear that a sequence enables Postgres to stream. This is connective tissue to tie the idea of a sequence to the capability of streaming

File: docs/how-sequin-works.mdx [hunk 2]
```diff
@@ -249,18 +249,53 @@ create table your_event_log (
...
-- `source_table_oid`: The OID for the source table.
-- `source_table_schema`: The schema of the source table.
-- `source_table_name`: The name of the source table.
-- `record_pk`: The primary key for the source row. It's `text`, regardless of the type of the source's primary key type.
-- `record`: For inserts and updates, this contains all the latest field values for the row (i.e. `new`). For deletes, this contains all the field values prior to deletion (i.e. `old`).
-- `changes`: For updates, this is a JSON of all the `old` fields _that changed_ in this update. `changes` does not include unchanged values. So, to get the entire `old` row, just merge `changes` on top of `record`. (`null` for insert and upate operations.)
-- `action`: One of `insert`, `update`, or `delete`.
-- `committed_at`: The time the change was committed.
-- `inserted_at` The time this event was inserted into the event table.
+<ResponseField name="id" type="integer" required>
```
Comment (thisisgoldman) [inline docs/how-sequin-works.mdx:252]: Formatting change from a list to the component

File: docs/introduction.mdx [hunk 3]
```diff
@@ -1,11 +1,13 @@
 ---
 title: What is Sequin?
-description: Start streaming Postgres in under five minutes
+description: Learn how to get up and running with Sequin.
 icon: "book-open"
 iconType: "solid"
 ---
 
-Sequin is a tool for capturing changes and streaming data out of your Postgres database. You can use Sequin to setup **HTTP endpoints** that you can use to stream rows from your database in real-time. Or, you can receive rows as they change via **webhooks**.
+Sequin is a tool for capturing changes and streaming data out of your Postgres database.
```
Comment (thisisgoldman) [inline docs/introduction.mdx:8]: My attempt at connecting "How Sequin Works" to "What is Sequin"

File: docs/introduction.mdx [hunk 4]
```diff
@@ -1,11 +1,13 @@
 ---
 title: What is Sequin?
-description: Start streaming Postgres in under five minutes
+description: Learn how to get up and running with Sequin.
```
[the remark this answers is not in the document]
Comment (acco) [reply docs/introduction.mdx:3]: I'm following Mintlify's own style here - no periods

I'm down to change the description, but I don't think "Learn how to get up and running with Sequin" is right

File: docs/images/core/consumer-workflow-diagram.png
Comment (acco) [inline docs/images/core/consumer-workflow-diagram.png:1]: Can you put Sync API on here so we don't need to redo this? It will be the same as the others

**Pull request merged**